src/_data/how_to/get_connected.yml

```diff
@@ -23,6 +23,37 @@ toc:
+        paragraphs:
+          - text: The FirstNet network was created and is sustained through a contract with the Federal Government. It is designed to offer a dedicated network for approved first responders. In California, transit providers are considered first responders and can access the FirstNet network.
+          - text: FirstNet is different from other cellular networks in that it provides priority for first responders automatically. This limits the potential for you to lose critical service capabilities during demanding times. In comparison to local emergency responder networks, FirstNet provides broadband voice, data, and video capabilities whereas typical emergency responder networks provide voice services.
+      - title: Why do the FirstNet plans have tiers? What do these tiers mean?
+        url: "firstnet-tiers"
+        paragraphs:
+          - text: Cal-ITP worked with FirstNet to create data plans tailored for transit agencies at a competitive price. The tiers incentivize more collective purchases and were designed by FirstNet to enable discounted rates in a way that works for their business case.
+          - text: The FirstNet tier prices are based on bulk volume purchased on that contract vehicle, (i.e., total number purchased by all agencies combined, not the number of SIMs purchased by an individual agency). Tier II pricing is guaranteed through March 2024. When the Tier III threshold is reached, all agencies will see their plans further discounted.
+      - title: Can FirstNet be used for any device and use case?
+        url: "firstnet-devices-use-case"
+        paragraphs:
+          - text: FirstNet only works for operations facing use cases, given it is a first responder network. This means it cannot be used for passenger Wifi. Additionally, devices are required to have an additional security certification to operate on the federal network. This certification is common but not universal and should be confirmed before purchase.
+          - text: A FirstNet representative can verify eligibility for your specific use cases and also confirm network compatibility with your devices. You can contact them at <a href="mailto:rc628k@att.com">rc628k@att.com</a> and <a href="mailto:bp1401@att.com">bp1401@att.com</a>.
+      - title: My agency is interested in purchasing a FirstNet data plan. How long will a purchase take?
+        url: "how-long-will-a-purchase-take"
+        paragraphs:
+          - text: The <a href="https://drive.google.com/file/d/1STiB4FsIZPTR97use5s9kdZAtWwdjE4l/view" target="_blank">purchase timeline</a> is anticipated to take anywhere from 10 to 14 weeks, assuming that the start date begins with the Official Notice of Receipt and concludes with SIM delivery.
+          - text: The installation timeline will vary based on if the agency or subcontractor will be responsible for installing the SIMs.
+      - title: Where can I access the non-FirstNet data plan rates in the table below?
```

Comment:
@mrose914 On the doc, it says "rates in the table above", but I changed it to "below" because the table is below this.